### DISCUSSION TITLE
Help:The performance of calling the API interface exposed through the workflow defined by dify is only half of the performance of directly calling the Chat interface.

### DISCUSSION THREAD

**Discussion by qaz163wsx** [langgenius/dify#6936] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I plan to use the differentiate workflow function to determine whether the content belongs to a certain category. For example, I want to determine whether an article talks about AI governance. I use the LLM node of the workflow to make a custom prompt determination. Because I have many articles and need to be called in large batches，but the current performance is only half that of calling the LLM API interface natively.

### 2. Additional context or comments

The workflow I tested only has one start node, one LLM judgment node, and one end node.

**Comment by Halfknow** [comment]
set auto_generate_name as false can fix the problem

**Reply by ee-redbull** [reply]
where to set?
